### ISSUE TITLE
ADDITIONAL_FOLDERS environment in docker-compose not working

### ISSUE THREAD

**Issue by jpdsc** [Minituff/nautical-backup#141]
Heya,

When adding:
```yaml
      ADDITIONAL_FOLDERS: "test"
      ADDITIONAL_FOLDERS_WHEN: "after"
```

to docker-compose.yml, it doesn't pick up the folder test. Instead it picks up -ahq. See DEBUG log below.

```
DEBUG: Backing up additional folder -ahq as it's in the ADDITIONAL_FOLDERS list.
DEBUG: RUNNING: 'rsync   /app/source/-ahq/ /app/destination/-ahq/'
rsync: [sender] change_dir "/app/source/-ahq" failed: No such file or directory (2)
rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1336) [sender=3.2.7]
INFO: Backed up additional folder '-ahq'
```

The folder test exists and using `INFO: Nautical Backup Version: 1.3.11`

**Comment by Minituff** [comment]
Oh that is weird. 


Are you able to post more of your Docker Compose? I can try to reproduce it on my end. 

Some quick troubleshooting ideas:

- Does this still happen when you change the `ADDITIONAL_FOLDERS_WHEN` to something else?
- Could you try removing the quotes in that part of the compose file? Like in the [example](https://minituff.github.io/nautical-backup/arguments/#additional-folders)? Maybe it's getting interpreted weird?

**Comment by jpdsc** [comment]
I did further analysis and using a different value in `ADDITIONAL_FOLDERS_WHEN` in this case, "before", works as intended.
The issue seems to be related in the "after" value.

compose:

```
  nautical-backup:
    container_name: nautical-backup
    image: minituff/nautical-backup:1
    restart: unless-stopped
    hostname: nas-1
    networks:
      - private
    ports:
      - 8069:8069
    environment:
      LOG_LEVEL: "DEBUG"
      TZ: "Europe/Amsterdam"
      HTTP_REST_API_ENABLED: "true"
      HTTP_REST_API_USERNAME: "admin"
      HTTP_REST_API_PASSWORD: "test"
      CRON_SCHEDULE: "false"
      REPORT_FILE: "true"
      REQUIRE_LABEL: "true"
      ADDITIONAL_FOLDERS: "test"
      ADDITIONAL_FOLDERS_WHEN: "before"
      USE_DEFAULT_RSYNC_ARGS: "true"
    volumes:
      - "/volume2/docker/appdata:/app/source:ro"
      - "/volume2/docker/dest-test:/app/destination"
      - "/var/run/docker.sock:/var/run/docker.sock"
      - "/etc/localtime:/etc/localtime:ro"
    labels:
      com.centurylinklabs.watchtower.enable: "true"
```

**Comment by Minituff** [comment]
Hi @jpdsc,

This should be fixed in a new PR I created here: https://github.com/Minituff/nautical-backup/pull/146.
I published a new [release ](https://github.com/Minituff/nautical-backup/releases/tag/v1.4.0)that should have this fix added in.

Thanks for finding this and doing some investigation to help me narrow it down--that made fixing it pretty straightforward.


---

Also, looking at your config. I'd love if you could explain:

```yml
CRON_SCHEDULE: "false"
volumes:
   - "/etc/localtime:/etc/localtime:ro"
 ```

I've never seen `/etc/localtime` mounted into the container before. What does this do? This could be something I could add to the docs if it's really useful.

Thanks

**Comment by jpdsc** [comment]
> Hi @jpdsc,
> 
> This should be fixed in a new PR I created here: #146. I published a new [release ](https://github.com/Minituff/nautical-backup/releases/tag/v1.4.0)that should have this fix added in.
> 
> Thanks for finding this and doing some investigation to help me narrow it down--that made fixing it pretty straightforward.
> 
> Also, looking at your config. I'd love if you could explain:
> 
> ```yaml
> CRON_SCHEDULE: "false"
> volumes:
>    - "/etc/localtime:/etc/localtime:ro"
> ```
> 
> I've never seen `/etc/localtime` mounted into the container before. What does this do? This could be something I could add to the docs if it's really useful.
> 
> Thanks

Good to read the issue is fixed :-).

Regarding the `"/etc/localtime:/etc/localtime:ro"` volume mount.
If a container does not have TZ env to determin timezone; I use `"/etc/localtime:/etc/localtime:ro"` to pass it from the host.

You actually noticed an issue in my compose file, this volume isn't applicable as you do have TZ env.

**Comment by Minituff** [comment]
Awesome. That's a great tip I'll save for whenever I'm working on containers without a TZ feature.

I'm going to close this issue, but but if you run into any other problems. Feel free to re-open/open a new issue.
